src/portable/espressif/esp32s2/dcd_esp32s2.c

```diff
@@ -127,7 +127,7 @@ static void bus_reset(void)
   //   Recommended value = 10 + 1 + 2 x (16+2) = 47 --> Let's make it 52
   USB0.grstctl |= 0x10 << USB_TXFNUM_S; // fifo 0x10,
   USB0.grstctl |= USB_TXFFLSH_M;        // Flush fifo
-  USB0.grxfsiz = 52;
+  USB0.grxfsiz = 15 + 2 * (256/4) + 2*EP_MAX;
```

Comment:
can you please explain this calculation?

Reply:
It's a workaround provided by @hathach in #540 , but actually need better solution.

Reply:
this is based on recommended by stm32 guide, however, it is not optimal for non-ISO application. I plan to work on a better fifo allocation scheme later on. I think it is probably best to leave this PR pending until we get thing settled with stm32 first. 2 moving targets are harder to work with than 1.